llm_transparency_tool/models/hash_funcs.py

```diff
@@ -0,0 +1,9 @@
+import transformers
+
+# If you load a new model from _model_path, please add its class here so that Streamlit can cache it
+
+HASH_FUNCS = {
+    transformers.PreTrainedModel: id,
+    transformers.PreTrainedTokenizer: id,
+    transformers.GPT2LMHeadModel: id,
+}
```

Comment:
I also found this limitation annoying, this means we have to run into hash function error then add them back to here.  Which means this list will getting longer and longer when supporting more models, even for models loaded from local directory.